docs/ingestion/ingestion-spec.md

```diff
@@ -475,10 +475,8 @@ For more details, see the documentation provided by each [ingestion method](./in
 
 ## `tuningConfig`
 
-Tuning properties are specified in a `tuningConfig`, which goes at the top level of an ingestion spec. Some
-properties apply to all [ingestion methods](./index.md#ingestion-methods), but most are specific to each individual
-ingestion method. An example `tuningConfig` that sets all of the shared, common properties to their defaults
-is:
+You specify tuning properties in a `tuningConfig`, which goes at the top level of an ingestion spec.
+Some properties apply to all [ingestion methods](./index.md#ingestion-methods), but most are specific to each individual ingestion method. An example `tuningConfig` that sets all of the shared, common properties to their defaults is:
```

Comment:
I think it'll be helpful to explain the structure of an ingestion spec and what each component of the spec is. (Doesn't have to be this PR)

1. Type
2. Data schema
2a. Dimension spec
2b. Metrics spec
3. ioConfig
3a. inputSource
3b. inputFormat
4. tuningConfig

etc.
(Include things like transform specs etc. so that people can see the full structure is helpful)
Here we can explain to people that tuningConfig is about things that affects ingestion performance, hardware resource utilization, as well as changing storage behavior. etc.